test/brsTypes/components/RoByteArray.test.js

```diff
@@ -0,0 +1,349 @@
+const os = require("os");
+const brs = require("../../../lib");
+const { RoArray, RoByteArray, BrsBoolean, BrsString, Int32, BrsInvalid } = brs.types;
+const { Interpreter } = require("../../../lib/interpreter");
+const { createMockStreams } = require("../../e2e/E2ETests");
+
+describe("RoByteArray", () => {
+    describe("comparisons", () => {
+        it("is equal to nothing", () => {
+            let a = new RoArray([]);
+            expect(a.equalTo(a)).toBe(BrsBoolean.False);
+        });
+    });
+
+    describe("stringification", () => {
+        it("lists values", () => {
+            let arr = new RoByteArray(new Uint8Array([1, 2, 3, 4, 5]));
+            expect(arr.toString()).toEqual(
+                `<Component: roByteArray> =
+[
+    1
+    2
+    3
+    4
+    5
+]`
+            );
```

Comment:
Might I suggest using a package like `undent` to allow these tests to read a little nicer? We do this a lot in brighterscript-formatter. (see [here](https://github.com/rokucommunity/brighterscript-formatter/blob/b634f670920cd7a7d78e8cede8a4f0e62e7871b4/src/Formatter.spec.ts#L18-L41)). 

So your test would look something like this:

```typescript
it("lists values", () => {
    let arr = new RoByteArray(new Uint8Array([1, 2, 3, 4, 5]));
    expect(arr.toString()).toEqual(`
        <Component: roByteArray> =
        [
        1
        2
        3
        4
        5
        ]
    `);
});
```

I feel like it improves the readability of the tests.  Maybe that could be a separate PR though to hit all the spots that do tests like this? Anyway, just a thought.